Lombiq.HelpfulLibraries.Cli/Lombiq.HelpfulLibraries.Cli.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Helpful Libraries - Command Line Libraries</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2011, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Helpful Libraries - Command Line Libraries. This project helps with executing command line calls.</Description>
```

Comment:
According to our [wiki](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution#:~:text=suffix%20it%20with%20%E2%80%9Csee%20the%20project%20website%20for%20detailed%20documentation.%E2%80%9D), the `Description` should end with:
```suggestion
    <Description>Lombiq Helpful Libraries - Command Line Libraries. This project helps with executing command line calls. See the project website for detailed documentation.</Description>
```